Transformers/Java/src/test/java/com/github/ciselab/lampion/transformations/RenameVariableTransformerTests.java

```diff
@@ -0,0 +1,276 @@
+package com.github.ciselab.lampion.transformations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.github.ciselab.lampion.transformations.transformers.RenameVariableTransformer;
+import java.util.List;
+import java.util.function.Predicate;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import spoon.Launcher;
+import spoon.reflect.code.CtLocalVariable;
+import spoon.reflect.declaration.CtClass;
+import spoon.reflect.declaration.CtElement;
+import spoon.reflect.declaration.CtMethod;
+import spoon.reflect.declaration.CtVariable;
+
+/**
+ * This is the test suite for the RenameVariableTransformer class.
+ * In order to resolve the issue of a new variable beginning with the same letter as the old one (int a -> int abba),
+ * we used an extra space, "int a " with the contains method.
+ * This should resolve this issue and make sure that the tests are not flaky.
+ */
+public class RenameVariableTransformerTests {
+
+    @RepeatedTest(5)
+    void applyTwiceToMethodWithTwoLocalVariables_firstOneThenTwoAltered(){
```

Comment:
Test-Names should follow the pattern:
methodUnderTest_Configuration_Modifier_ExpectedResult

Then the final test reports are quite verbatim and seeing which ones go red is quite expressive.